_std/defines/bool.dm

```diff
@@ -0,0 +1,2 @@
+#define true 1
```

Comment:
better done in a `logic.dm` so it can incorporate more general stuff, or moved to `math.dm` imo

Reply:
renamed to logic.dm, im not sure how i feel about true/false defines being in math.dm